fortis-deploy.sh

```diff
@@ -146,7 +146,9 @@ azure_login() {
 install_helm() {
   curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
   chmod 700 get_helm.sh
-  ./get_helm.sh
+  readonly helm_version="v2.5.1" 
+  ./get_helm.sh -v ${helm_version}
```

Comment:
Needs quotes around the variable to avoid whitespace expansion and lint errors: `./get_helm.sh -v "${helm_version}"`